proposals/lip-0051.md

```diff
@@ -2420,19 +2419,18 @@ def supportAllTokensFromChainID(chainID: ChainID) -> None:
 
 #### removeAllTokensSupportFromChainID
 
-This function is called in case where all tokens from a specific chain are supported and we want to remove this support. After calling this function no tokens of the specified chain are supported. 
-
+This function is called to remove support for all tokens of a specified chain. 
 
 ```python
 def removeAllTokensSupportFromChainID(chainID: ChainID) -> None:
     if there exists entry in the supported tokens substore with key == ALL_SUPPORTED_TOKENS_KEY:
-        return
+        raise Exception('Invalid operation. All tokens from all chains are supported.')
```

Comment:
i think this operation is not needed because if chainID doesn't exist in data, it will fail, and if it exist it never has ALL_SUPPORTED_TOKEN.

Or if we say
```
if supportedTokens(chainID) does not exist:
        return
```
as below, this should also return?